08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,31 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+var TimeConverter = React.createClass({
+    getInitialState: function() {
+        return {
+            hours : 1,
+            minutes : 60,
+            seconds : 3600
+        };
+    },
+    render: function() {
+        return (
+            <div>
+                <h6>Hours</h6>
+                <input type="text" value={this.state.seconds / 3600} onChange={this.handleChange.bind(this, 'hours')} />
+                <br/>

```

Comment:
I advice not to use <br/> tags. If you want proper separation between html elements better do it with css or use inline style with "style" prop.